app/lib/shared/convert.dart

```diff
@@ -0,0 +1,42 @@
+import 'dart:convert';
```

Comment:
I wish we had these in `dart:convert`.

Reply:
Yes, I considered adding them as a constructor...